### ISSUE TITLE
Linkchecker still points to old URL

### ISSUE THREAD

**Issue by a3nm** [linkchecker/linkchecker#110]
When running linkchecker, I still see pointers to the URL of the dead project:

```
LinkChecker 9.4              Copyright (C) 2000-2014 Bastian Kleineidam
LinkChecker comes with ABSOLUTELY NO WARRANTY!
This is free software, and you are welcome to redistribute it
under certain conditions. Look at the file `LICENSE' within this
distribution.
Get the newest version at http://wummel.github.io/linkchecker/
Write comments and bugs to https://github.com/wummel/linkchecker/issues
Support this project at http://wummel.github.io/linkchecker/donations.html
```

I guess these links should be to the new URL instead.

**Comment by anarcat** [comment]
i've created issue #136 to track those kind of issues more globally.

**Comment by cjmayo** [comment]
This was fixed in 004632a99b by #242.
